fetch/corb/style-css-mislabeled-as-html-nosniff.sub.html

```diff
@@ -31,6 +31,7 @@ <h1 id="header">Header example</h1>
 </body>
 
 <script>
+setup({ single_test: true });
```

Comment:
@jugglinmike I see that this and a few of the following are just sync tests, can you wrap them in `test(() => { ... })` instead?

Reply:
Sure. I've added titles to those tests, as well